ansible_collections/arista/avd/roles/eos_designs/templates/network_services/vlan-interfaces.j2

```diff
@@ -58,8 +58,8 @@ vlan_interfaces:
     ip_address_virtual_secondaries: {{ svi.ip_address_virtual_secondaries }}
 {%             endif %}
 {# Virtual Secondary IPv6 address #}
-{%             if svi.ipv6_address_virtual_secondaries is arista.avd.defined %}
-    ipv6_address_virtual_secondaries: {{ svi.ipv6_address_virtual_secondaries }}
+{%             if svi.ipv6_address_virtuals is arista.avd.defined %}
+    ipv6_address_virtuals: {{ svi.ipv6_address_virtuals }}
```

Comment:
I cannot see this key in the `eos_cli_config_gen` README - only `ipv6_address_virtual` https://github.com/aristanetworks/ansible-avd/pull/2141/files#diff-6519adf3b51d9eb64589d80a581782a31def7cad2ae99893e6bcd3adb5ed4e54R1381 

It is in the template as introduced in this PR.

Reply:
added